src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -929,6 +941,14 @@ public function __call($method, $parameters)
 		{
 			return $this->callScope($scope, $parameters);
 		}
+		elseif ( ! is_null($this->model['fillable']) && in_array($column = 'is_'.$method, $this->model['fillable']))
+		{
+			return $this->addBooleanColumnScope($column, true);
+		}
+		elseif ( ! is_null($this->model['fillable']) && in_array($column = 'is_'.lcfirst(str_replace('not', '', $method)), $this->model['fillable']))
```

Comment:
That is a very valid point, and a good reason why I shouldn't make pull requests early in the morning. I will adjust the code to only remove the first instance of 'not' and squash again